src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -355,5 +355,21 @@ def pick_max_problems(remaining_checks, remaining_build_problems, max_shown)
     build_problems = remaining_build_problems.shift(show_builds)
     return checks, build_problems, remaining_checks, remaining_build_problems
   end
+
+  # responsive_ux: used only in beta feature
+  def home_project_link
+    return project_show_path(User.session!.home_project_name) if User.session!.home_project
```

Comment:
`User.session!` raises an exception for nobody...

Reply:
I also don't see you using this anywhere?

Reply:
Old code, we remove it.